api/app/Services/ModelFactoryInstance.php

```diff
@@ -248,6 +248,12 @@ public function transformed()
             }
 
             $this->setEntityType();
+        } elseif ($this->factoryInstance instanceof BaseModel) {
+            $entity = $this->factoryInstance;
```

Comment:
I'm not happy with this additional capability of the model factory service to be able to take existing models/collections. I've created #207 to refactor out the collection capability. Basically the intent of the model factory is to be able to generate mocks, not modify existing entities.

The usages of this feature can be refactored easily to not rely on this feature too, example:
```
        $tag = $this->getFactory()->get(\App\Models\Tag::class)->create();
        $tagTransformed = $this->getFactory()->get($tag)
            ->customize(['tag' => 'foo'])
            ->transformed();
```
Can become
```
        $tagFactory = $this->getFactory()->get(\App\Models\Tag::class);

        $original = $tagFactory->create();
        $modifiedTransformed = $tagFactory->customize(['tag' => 'foo'])
            ->transformed();
```